modules/genomic_browser/php/endpoints/uploading/genomicfile.class.inc

```diff
@@ -0,0 +1,652 @@
+<?php declare(strict_types=1);
+/**
+ * POST request method.
+ *
+ * Used for uploading file with Genomic Browser.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Genomic_Browser
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\genomic_browser\endpoints\uploading;
+
+/**
+ * The Genomicfile Class.
+ *
+ * This class provides the php code to process uploading
+ * a user file from the genomic browser.
+ *
+ * @category Loris
+ * @package  Genomic_Browser
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Genomicfile
```

Comment:
This whole uploading file process will be a lot easier if you use the File Uploader class from #4181 instead. Not a blocking change but this will need to be refactored to use it at some point anyway.

Reply:
@johnsaigle after this PR is merged I think that should be next step. This PR was urgent for removing the old way of filtering the table from LORIS.